Library/Homebrew/cmd/test-bot.rb

```diff
@@ -91,13 +92,31 @@ def failed?
 
     def puts_command
       cmd = @command.join(" ")
-      print "#{Tty.blue}==>#{Tty.white} #{cmd}#{Tty.reset}"
+      cmd_line = "#{Tty.blue}==>#{Tty.white} #{cmd}#{Tty.reset}"
+      if ENV["TRAVIS"]
+        @travis_timer_id = rand(2**32).to_s(16)
+        puts "travis_fold:start:#{@command.join(".")}"
+        puts "travis_time:start:#{@travis_timer_id}"
+        puts cmd_line
+        return
+      end
+      print cmd_line
       tabs = (80 - "PASSED".length + 1 - cmd.length) / 8
       tabs.times { print "\t" }
       $stdout.flush
     end
 
     def puts_result
+      if ENV["TRAVIS"]
+        cmd = @command.join(" ")
+        puts "#{Tty.send status_colour}==> #{cmd}: #{status_upcase}#{Tty.reset}"
```

Comment:
However, travis still hides the result. See https://travis-ci.org/Homebrew/homebrew/jobs/80865031